package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Calvin Montgomery",
   "name": "CyTube",
   "description": "Online media synchronizer and chat",
-  "version": "3.17.5",
+  "version": "3.17.6",
```

Comment:
I've been pretty bad about following this in the past, but I think that this particular change is worthy of bumping the minor version to 18.